code/modules/antagonists/gang/gang.dm

```diff
@@ -548,3 +550,184 @@
 	if(mat_container.materials[SSmaterials.GetMaterialRef(/datum/material/gold)] >= 2000) // if theres at least 1 bar of gold left in the silo, they've failed to heist all of it
 		return FALSE
 	return TRUE
+
+/datum/antagonist/gang/irs
+	show_in_antagpanel = TRUE
+	name = "Internal Revenue Service Agent"
+	roundend_category = "IRS Agents"
+	gang_name = "Internal Revenue Service"
+	gang_id = "IRS"
+	acceptable_clothes = list(/obj/item/clothing/suit/irs,
+							/obj/item/clothing/under/costume/irs,
+							/obj/item/clothing/head/irs)
+	free_clothes = list(/obj/item/clothing/suit/irs,
+							/obj/item/clothing/under/costume/irs,
+							/obj/item/clothing/head/irs,
+						/obj/item/toy/crayon/spraycan)
+	gang_objective = "TODO: Add Objective"
+	antag_hud_name = "IRS"
+
+/datum/antagonist/gang/irs/check_gang_objective()
+	return TRUE
+
+/datum/antagonist/gang/osi
+	show_in_antagpanel = TRUE
+	name = "Office of Secret Intelligence Agent"
+	roundend_category = "O.S.I. Agents"
+	gang_name = "Office of Secret Intelligence"
+	gang_id = "OSI"
+	acceptable_clothes = list(/obj/item/clothing/suit/osi,
+							/obj/item/clothing/under/costume/osi,
+							/obj/item/clothing/glasses/sunglasses/osi)
+	free_clothes = list(/obj/item/clothing/suit/osi,
+							/obj/item/clothing/under/costume/osi,
+							/obj/item/clothing/glasses/sunglasses/osi,
```

Comment:
damn, these guys get free sunnies? that certainly kicks them up the gang tier list; you might want to give them a slightly harder than normal (but NOT impossible) objective or something to compensate